expression/integration_test.go

```diff
@@ -2979,6 +2979,18 @@ func (s *testIntegrationSuite) TestIssues(c *C) {
 	r.Check(testkit.Rows("1 hello", "2 hello"))
 }
 
+func (s *testIntegrationSuite) TestNoUnsignedSubTractionMode(c *C) {
+   tk := testkit.NewTestKit(c, s.store)
+   defer s.cleanEnv(c)
+   tk.MustExec("use test")
+   tk.MustExec("drop table if exists t")
+   tk.MustExec("CREATE TABLE t (a BIGINT, b BIGINT UNSIGNED);")
```

Comment:
We don't need to create a table for this test.

And please add more tests like
`SELECT CAST(18446744073709551615 as UNSIGNED) - 1;`
`SELECT 1 - CAST(18446744073709551615 as UNSIGNED);`